apps/modernization-ui/src/apps/patient/data/identification/IdentificationEntryFields.tsx

```diff
@@ -0,0 +1,91 @@
+import { usePatientIdentificationCodedValues } from 'apps/patient/profile/identification/usePatientIdentificationCodedValues';
+import { DatePickerInput } from 'components/FormInputs/DatePickerInput';
+import { Input } from 'components/FormInputs/Input';
+import { SingleSelect } from 'design-system/select';
+import { Controller, useFormContext } from 'react-hook-form';
+import { maxLengthRule } from 'validation/entry';
+import { IdentificationEntry } from '../entry';
+
+export const IdentificationEntryFields = () => {
+    const { control } = useFormContext<IdentificationEntry>();
+
+    const coded = usePatientIdentificationCodedValues();
```

Comment:
question (non-blocking):  Do we want to continue using the lists resolved from graphql?  These are coded-values and should be available through `useConceptOptions`.

Reply:
Actually...we should leave this hook in place and just swap out the innards in the future.